vault/resource_pki_secret_backend_root_cert.go

```diff
@@ -53,9 +57,38 @@ func pkiSecretBackendRootCertResource() *schema.Resource {
 				return e
 			}
 
-			cert, err := getCACertificate(client, d.Get(consts.FieldBackend).(string))
-			if err != nil {
-				return err
+			var cert *x509.Certificate
+			isIssuerAPISupported := provider.IsAPISupported(meta, provider.VaultVersion111)
```

Comment:
Prior to multi-issuer support (<= Vault 1.10), this customize diff would read the default CA in PEM format and compare to what was in the TF state. When there is only one issuer, this makes sense since it will be the default. However, for multiple issuers, all Root Cert resources that are not the default issuer will be forced to be recreated. This was leading users into a constant create-destroy cycle when multiple root certs were added in a TF file.

This update now correctly read the specific issuer's certificate in PEM format, and compare to what is in the TF state. If there is a change in the certificate, the Root Cert is recreated. Also, if an issuer is deleted outside of TF, this update correctly resolves the state by recreating the resource.